### PR TITLE
User facing channel names swapped for bts channel names

### PR DESCRIPTION
**Pull request by derollins**
This means the reported available channels in the logger error message are actually what you need to put into the config file.

### CODE REVIEW

**Review by MaxGamill-Sheffield — APPROVED** [review]
Looks good and tested with channel description outputs from my a cat image.

File: topostats/io.py
Comment (ns-rse) [inline topostats/io.py:500]: Do we need to retain this line if the correct value should always be `channel_description`?

If not this line can be removed.

The code will still be present in the Git history and referenceable by looking at the Git blame for the line that remains, tracing it back to the commits and then its possible to see what changed if the old value is required.

**Review by ns-rse — APPROVED** [review]
Looks good and thanks for getting stuck in and making your first PR @derollins .

On reading this I wondered if this might help with the issue #619 and having seen @MaxGamill-Sheffield comment it seems it is.

File: topostats/io.py
Comment (ns-rse) [inline topostats/io.py:500]:
```suggestion

```

Comment (ns-rse) [reply topostats/io.py:500]: Ok, two approaches @derollins 

1) You can if you want directly commit the above suggestion to this Pull Request. The Pull Request has been approved already, this may remove the approval if so it will require approving again. This change won't be reflected on your local branch. That isn't necessarily a problem though as you probably won't touch this branch again after its merged and can delete it.

2) Make the change on your local branch (on your laptop) and push it to GitHub, it will be tagged onto and included in this Pull Request. I would expect this to remove the approval too and it would then require re-approval.

Once re-approvals have been made you can go ahead and "Merge" the pull request, on doing so you'll be offered the opportunity to delete the branch on GitHub which is perfectly fine to do.

**Review by ns-rse — COMMENTED** [review]
Made a suggestion about how to handle the line removal.

**Review by ns-rse — APPROVED** [review]
Brilliant, thanks @derollins this looks ready to merge now. :+1: 

/cc @neumagen this may solve your channel issue. You'll have to setup a new virtual(/conda) environment and install TopoStats from Git to try it out, but we have instructions on how to do this on the [website](https://afm-spm.github.io/TopoStats/main/installation.html#cloning-from-github).

If you encounter any problems you can ask for support in the [Installation](https://github.com/AFM-SPM/TopoStats/discussions/categories/installation) discussion category.

**Pull request merged**
